elfo-network/src/discovery.rs

```diff
@@ -311,6 +314,14 @@ async fn accept_connection(
         elapsed = ?start.elapsed(),
     );
 
+    if node_no == this_node.node_no {
+        debug!(
```

Comment:
I believe it should be `info` because "new connection established" is also info.